build.gradle

```diff
@@ -243,18 +243,18 @@ dependencies {
     exclude group: 'junit', module: 'junit'
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
-  testImplementation group: 'com.azure', name: 'azure-core-test', version: '1.1.0', {
+  testImplementation group: 'com.azure', name: 'azure-core-test', version: '1.2.1', {
     exclude group: 'com.github.tomakehurst', module: 'wiremock-standalone'
   }
 
   testCompile group: 'org.apache.commons', name: 'commons-email', version: '1.5'
 
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
-  integrationTestImplementation group: 'com.azure', name: 'azure-core-test', version: '1.1.0', {
+  integrationTestImplementation group: 'com.azure', name: 'azure-core-test', version: '1.2.1', {
```

Comment:
weird. shouldn't be here if it is in `testImplementation` scope

Reply:
I thought it is on purpose. removed it.